packages/locale/src/format/cldr/CldrCache.js

```diff
@@ -0,0 +1,54 @@
+/*
+* @copyright (c) 2020, Philipp Thuerwaechter & Pattrick Hueper
+* @license BSD-3-Clause (see LICENSE.md in the root directory of this source tree)
+*/
+
+import cldrData from 'cldr-data';
+import Cldr from 'cldrjs';
+
+const cldrDataLoaded = new Set();
+/**
+* Loads the Cldr data for the given path if it hasn't been loaded before.
+*/
+export const loadCldrData = (path) => {
+    if (!cldrDataLoaded.has(path)) {
+        Cldr.load(cldrData(path));
+        cldrDataLoaded.add(path);
+    }
+};
+
+const localeToCldrInstanceCache = {};
+/**
+* Returns a Cldr instance for the given locale.
+* Memoized, so a given locale will always return the exact same cldr instance.
+*/
+export const getOrCreateCldrInstance = (locale) => {
+    if (localeToCldrInstanceCache[locale] == null) {
+        localeToCldrInstanceCache[locale] = new Cldr(locale);
+    }
+    return localeToCldrInstanceCache[locale];
+};
+
+const cldrInstanceToMapZonesCache = new Map();
+/**
+* Returns a map zones object for a cldr instance.
+* Memoized, so a given cldr instance will always return the exact same object.
+*/
+export const getOrCreateMapZones = (cldr) => {
+    if (!cldrInstanceToMapZonesCache.has(cldr)) {
```

Comment:
Would it possible to change the key from a cldr instance to a locale string ? It is very hard to read and understand why this is mapped to an instance. A locale string would make this much more clear IMO.

Regarding cache size, again, some numbers would be great.

Reply:
Yes, should be easy to switch to make this take a locale instead! I'll do that